src/arm/windows/init.c

```diff
@@ -64,6 +79,32 @@ static struct woa_chip_info woa_chips[] = {
 				3150000000
 			}
 		}
+	},
+	/* Microsoft Windows Dev Kit 2023 */
+	{
+		"Snapdragon (TM) 8cx Gen 3 @ 3.0 GHz",
+		woa_chip_name_microsoft_sq_3,
+		{
+			{
+				cpuinfo_uarch_cortex_a78,
+				2420000000,
+			},
+			{
+				cpuinfo_uarch_cortex_x1,
+				3000000000
+			}
+		}
+	},
+	/* Ampere Altra */
+	{
+		"Ampere(R) Altra(R) Processor",
+		woa_chip_name_ampere_altra,
+		{
+			{
+				cpuinfo_uarch_cortex_a76,

```

Comment:
Per https://www.anandtech.com/show/16315/the-ampere-altra-review/2, Ampere Altra uses Neoverse-N1 cores. Should be `cpuinfo_uarch_neoverse_n1`

Reply:
You are right, thank you.